frontend/app/src/components/AppView/styled-components.ts

```diff
@@ -102,10 +105,28 @@ export interface StyledAppViewBlockContainerProps {
   isWideMode: boolean
   showPadding: boolean
   addPaddingForHeader: boolean
-  disableFullscreenMode: boolean
   hasBottom: boolean
 }
 
+/**
+ * Adds the CSS query for wide mode.
+ */
+const applyWideModePadding = (theme: EmotionTheme): CSSObject => {
+  return {
+    // Increase side padding, if the current container width is greater than the contentMaxWidth
+    // + the wide mode padding - the the centered padding (lg). This allows a smooth transition
+    // when switching between centered to wide mode when resizing the window.
+    // And makes sure that wide mode always has the same or larger width than centered mode.
+    [`@container (min-width: calc(${theme.sizes.contentMaxWidth} + 2 * (${theme.sizes.wideSidePadding} - ${theme.spacing.lg})))`]:
```

Comment:
Container queries have quite good support across major browsers, but some of the less common mobile OS browsers do not support it yet: https://caniuse.com/css-container-queries

The effect would be that these browsers would just show it in the default-centered mode which is mostly the same anyways on small screen sizes.